docs/build.md

```diff
@@ -69,6 +70,20 @@ curl -fsSL -o libsodium-src.tar.gz "$LIBSODIUM_DOWNLOAD_URL" \
 && cd ..
 ```
 
+## MacOS
+
+Install `brew` if not done yet:
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+Also, install dependencies:
+```
+brew update
+brew install erlang
```

Comment:
I think this will install 21.3.2 which is not supported by the node yet. @lucafavatella Do we support 21 for builds or we don't support it runtime only ?

Reply:
Building with 21 succeeded on my side.

Reply:
Let's double check with @lucafavatella

Reply:
Our CI checks that [building with OTP 21 works for the purpose of running some static checks](https://github.com/aeternity/aeternity/blob/27f5df09a7bb70dc2a43c29106acc1b56ff1c303/.circleci/config.yml#L474). We do not support at runtime only e.g. re logging behaviour.

Reply:
So we're OK to publish build docs for OTP 21 ? (default by brew install)

Reply:
I am considering about support for such documented instructions. Considering that our CI tests build on macOS using our pinned formula (see [CI config](https://github.com/aeternity/aeternity/blob/27f5df09a7bb70dc2a43c29106acc1b56ff1c303/.circleci/config.yml#L571)), may I suggest to replace this with something like:
```suggestion
brew install aeternity_source/deployment/homebrew/erlang.rb
```
Though this ^^^ may benefit from being better integrated in the document (as the cloning of the repo is documented below this).